aeron-cluster/src/main/java/io/aeron/cluster/ClusterBackupAgent.java

```diff
@@ -306,6 +310,19 @@ private void reset()
         CloseHelper.close(multiSnapshotReplication);
         multiSnapshotReplication = null;
 
+        if (NULL_VALUE != liveLogReplaySessionId)
+        {
+            try
+            {
+                clusterArchive.stopReplay(liveLogReplaySessionId);
```

Comment:
This will not work. You need the full replay session id and not just the lower bits for the stream session.